ios/MagnetScannerReact.swift

```diff
@@ -26,39 +23,6 @@ import Foundation
       .sendDeviceEventWithName("magnetscanner:itemfound", body: item)
     
     let url = item["url"] as! String
-    notifyUser(url)
```

Comment:
Moved all this code to notifications, this was in charge of throttling  the notifications, so is better to live there, also used now for the geo notifications in bg.